test/format/ops.c

```diff
@@ -230,12 +230,20 @@ tinfo_teardown(void)
 static void
 rollback_to_stable(WT_SESSION *session)
 {
+    u_int num_threads;
+    char cfg[512];
```

Comment:
Do we need this big array size?

Reply:
It feels like 32 would be sufficient.

Reply:
Updated to 32 in 81a32448f1615b184695befd4d7f16a61b2c4637